firefox_decrypt.py

```diff
@@ -768,6 +779,10 @@ def parse_sys_args():
 
     args = parser.parse_args()
 
+    if args.tabular:
+        args.format = 'csv'
+        args.delimiter = '\t'
```

Comment:
Missing args.quotechar = "'" to ensure backwards compatibility.

I'm actually not entirely sure what the csv module does if there's a ' symbol in a password. It might escape it somewhow. A test should also cover this case.